src/openeo_grass_gis_driver/actinia_processing/base.py

```diff
@@ -74,12 +75,14 @@ def __init__(
             name: str,
             datatype: GrassDataType,
             mapset: str = None,
-            location: str = None):
+            location: str = None,
+            instance: str = None):
 
         self.name = name
         self.datatype = datatype
         self.mapset = mapset
         self.location = location
+        self.instance = instance
```

Comment:
Would it be possible to add the instance at a later step? Or as it is set below as mapset anyway - couldn't the value of mapset be used directly? If not, please add a comment above that instance is only needed for STAC in actinia because of the concept that a catalog must belong to an instance.